tests/conftest.py

```diff
@@ -62,108 +160,104 @@ def bytes_helper(str, length):
 
 
 @pytest.fixture
-def t():
-    tester.s = tester.Chain()
-    return tester
+def get_contract_from_lll(w3):
+    def lll_compiler(lll, *args, **kwargs):
+        lll = optimizer.optimize(LLLnode.from_list(lll))
+        bytecode = compile_lll.assembly_to_evm(compile_lll.compile_to_assembly(lll))
+        abi = []
+        contract = w3.eth.contract(bytecode=bytecode, abi=abi)
+        deploy_transaction = {
+            'data': contract._encode_constructor_data(args, kwargs)
+        }
+        tx = w3.eth.sendTransaction(deploy_transaction)
+        address = w3.eth.getTransactionReceipt(tx)['contractAddress']
+        contract = w3.eth.contract(address, abi=abi, bytecode=bytecode, ContractFactoryClass=ConciseContract)
```

Comment:
I see you are using `ConciseContract` copiously.  My first inclination was to push you away from it, but I think that's mostly my personal preference speaking.

Was the choice to use this happenstance?  Intentional over the more explicit call semantics for the default `Contract` class?